repo/meta/schema/resource-schema.json

```diff
@@ -1,55 +1,147 @@
 {
-  "$schema": "http://json-schema.org/schema#",
-  "type": "object",
-  "properties": {
-    "assets": {
-      "type": "object",
-      "properties": {
-        "uris": {
-          "type": "object",
-          "additionalProperties": {
-            "type": "string"
-          }
+    "$schema": "http://json-schema.org/schema#",
```

Comment:
The schema value should be `http://json-schema.org/draft-04/schema#` when using document refs